### PR TITLE
feat: add scraper for DEU (Germany)

### PR DESCRIPTION
**Pull request by jgehrcke**
Hey!

The README patch is just a minor proposal, feel free to say if you don't like that.

I have tried to test the scraper:
```
$ yarn start --location "DEU"
yarn run v1.22.4
$ NODE_OPTIONS='--insecure-http-parser' node cli.js --location DEU
(node:15641) ExperimentalWarning: The ESM module loader is experimental.
⏳ Scraping data for today...
  🐢 Cache miss for https://covid19-germany.appspot.com/now at coronadatascraper-cache/2020-3-17/38f8f7262afb9fab969c26940d40bca8.json
  🚦  Loading data for https://covid19-germany.appspot.com/now from server
(node:15641) Warning: Using insecure HTTP parsing
  ⚡️ Cache hit for https://opendata.arcgis.com/datasets/d14de7e28b0448ab82eb36d6f25b1ea1_0.csv from coronadatascraper-cache/2020-3-17/ea58672b23e340fbf8f209b7af40173c.csv
...
```
Seems to have worked in the sense that it did not error out.

### CODE REVIEW

**Comment by jgehrcke** [comment]
As for the data source https://covid19-germany.appspot.com/now also see https://github.com/lazd/coronadatascraper/issues/79.

The data source is served by a Google App Engine instance which returns a JSON document of the following structure:
```
$ curl https://covid19-germany.appspot.com/now 2> /dev/null| jq
{
  "current_totals": {
    "cases": 9348,
    "deaths": 25,
    "recovered": 72,
    "tested": "unknown"
  },
  "meta": {
    "contact": "Dr. Jan-Philip Gehrcke, jgehrcke@googlemail.com",
    "source": "zeit.de (aggregates data from individual ministries of health in Germany)",
    "time_source_last_consulted_iso8601": "2020-03-18T00:11:24+00:00",
    "time_source_last_updated_iso8601": "2020-03-17T21:22:00+01:00"
  }
}
```


- We have the challenge here in Germany that the country-level institute (Robert Koch-Institut) is rather slow collecting and re-publishing data (~1-2 days).
- Some individual ministries of health of the individual countries publish their data online in various forms (I tried following this tail), but not all of them. Some ministries seem to communicate their numbers when asked e.g. via phone.
- As of today, one of the most serious and biggest newspapers in Germany, zeit.de, seems to be doing the best job of collecting the individual counts from the various ministries of health.
- The GAE instance I have stood up (code [here](https://github.com/jgehrcke/covid-19-germany-gae)) serves as an interface proxy and cache, yielding a stable interface to consumers like this project here.

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ If you've already cloned without `--recursive`, run:
 ```
 git submodule init
 git submodule update
+cd coronavirus-data-sources && git submodule init && git submodule update
```
Comment (lazd) [inline README.md]: Does this fix all the dang errors we've been having with submodules?!

**Comment by jgehrcke** [comment]
>  Does this fix all the dang errors we've been having with submodules?!

Can't comment on that, but I've documented in the corresponding commit message what it fixed for me!

File: README.md (hunk 1 above)
Comment (lazd) [reply README.md]: Hmm ok, but I wonder, can we pass `--recursive` to `git submodule init`? or maybe do `git submodule update --init --recursive`? Can you try fresh and see what works and doesn't require `cd`ing?

Comment (jgehrcke) [reply README.md]: I have removed the README patch from this branch for now.

>  Can you try fresh and see what works and doesn't require cding?

Will do and report back!

Comment (qgolsteyn) [reply README.md]: In our CI scripts, we `git submodule update --init --recursive` which has been working well so far. Big issue is the submodule inside `coronavirus-data-sources` hence why we need to add `--recursive`

**Comment by qgolsteyn** [comment]
Awesome work, thanks!

**Comment by jgehrcke** [comment]
Thanks!

**Comment by Wikunia** [comment]
Is there a way to have state level data for Germany?

**Pull request merged**